languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -1073,6 +1073,76 @@ USA
             <example correction="we're">Trying to figure-out if <marker>were're</marker> going to Swansboro</example>
             <example correction="We're"><marker>Were're</marker> making a big deal out of nothing.</example>
         </rule>
+        <rule id="HAS_IT_NNS__IT_ITS" name="has it advantages → has its advantages" default="temp_off">
+            <antipattern><!-- There we have it -->
+                <token>there</token>
+                <token postag="PRP">
+                    <exception case_sensitive="yes" regexp="yes">[Ii]t</exception>
+                </token>
+                <token>have</token><!-- TODO: inflected? -->
+                <token>it</token>
+            </antipattern>
+            <!-- Copied antipattern from MAKE_PRP_VBZ -->
+            <antipattern><!-- For every room I have it bundles all lights. -->
+                <token>for</token>
+                <token skip="-1" postag="DT" regexp="yes">each|every|all|both|most|some|either|neither</token>
+                <token postag_regexp="yes" postag="VBN?" />
+                <!-- Ideally insert a comma -->
+                <token postag="PRP" />
+            </antipattern>
+
```

Comment:
There's a `TODO` in there, is the rule ready to be merged yet?
Could you remove the empty lines, please?